SU-HSDO/suhumsci
# STN-623  Bug fix dark masthead pipe

# READY FOR REVIEW

## Summary
Dark Masthead causing pipes issue on the lockup. This should be updated for both themes and color palettes.

![image-2020-11-06-14-54-31-909](https://user-images.githubusercontent.com/9355698/108661696-7617f800-749a-11eb-89c2-365b662625a6.png)

![image](https://user-images.githubusercontent.com/9355698/108661760-a495d300-749a-11eb-8e6e-87cab25e9763.png)

[JIRA Card: STN-623](https://sparkbox.atlassian.net/browse/STN-623)

## Need Review By (Date)
2/22/21

## Urgency
medium

## Steps to Test
1. npm run test
2. Switch the masthead over to Dark in the Theme Settings.
3. For the vertical pipe bug, you can zoom your browser to see the pipes when not overlaying each other, however, all pipes should be white and no longer have a half of a pipe as a darker tertiary color.
4. To check the Horizontal pipe in the lockup, swap the lockup over to Option S or T in the theme setting for the lockup option.
**You may have to fill out some additional site lockup text options to see the horizontal bar appear.**
It should also now be white like the vertical pipe and text in the dark masthead option.
![image](https://user-images.githubusercontent.com/9355698/108661516-0144be00-749a-11eb-8cc6-9ae178da4aae.png)
5. Verify there is no impact or changes to the light masthead variant.
6. Verify this works correctly on both themes and color pairings.



## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
